apps/sap-commerce-cloud/frontend/package.json

```diff
@@ -23,7 +23,9 @@
     "start": "vite",
     "build": "vite build",
     "test": "vitest --logHeapUsage --coverage",
-    "test:ci": "vitest"
+    "test:ci": "vitest",
+    "lint": "eslint 'src/**/*.{js,jsx,ts,tsx}'",
```

Comment:
In several other apps, we have the `--max-warnings 0` option so that it will fail on CI if there are any warnings. Thoughts on adding that?

Reply:
great idea, thanks!